feature/achievements/src/main/java/io/github/droidkaigi/confsched2023/achievements/AchievementsScreen.kt

```diff
@@ -83,51 +90,85 @@ fun AchievementsScreen(
         snackbarHostState = snackbarHostState,
         contentPadding = contentPadding,
         onReset = viewModel::onReset,
+        showAnimation = { achievement -> viewModel.onClickAchievement(achievement) },
+        finishAnimation = viewModel::onFinishAnimation,
```

Comment:
We would like to promote the best practice that a StateHolder should be notified events. 
So how about naming this `onAchivementClicked` and judge if we run a animation in ViewModel for `showAnimation`?
And use "onAnimationFinished" as a name for `finishAnimation`?

Reply:
Renamed showAnimation to onAchivementClick and finishAnimation to onAnimationFinish.